### ISSUE TITLE
problem with treeview async

### ISSUE THREAD

**Issue by gabrielrangelm** [f4bsch/WP-Filebase#59]
Hi, I've this problem with the treeview_

Do not expand the tree, this is the jquery error:

jquery.js?ver=1.12.4:3 Uncaught TypeError: Cannot read property 'createDocumentFragment' of undefined
    at ca (http://172.20.10.2/portal/wp-includes/js/jquery/jquery.js?ver=1.12.4:3:4983)
    at ja (http://172.20.10.2/portal/wp-includes/js/jquery/jquery.js?ver=1.12.4:3:7042)
    at Ha (http://172.20.10.2/portal/wp-includes/js/jquery/jquery.js?ver=1.12.4:3:20877)
    at a.fn.init.append (http://172.20.10.2/portal/wp-includes/js/jquery/jquery.js?ver=1.12.4:3:22791)
    at a.fn.init.n.fn.(anonymous function) [as appendTo] (http://172.20.10.2/portal/wp-includes/js/jquery/jquery.js?ver=1.12.4:3:24510)
    at Object.createNode (http://172.20.10.2/portal/wp-content/plugins/wp-filebase/extras/jquery/treeview/jquery.treeview.async.js?ver=0.3.0.06:15:93)
    at Function.each (http://172.20.10.2/portal/wp-includes/js/jquery/jquery.js?ver=1.12.4:2:2881)
    at Object.success (http://172.20.10.2/portal/wp-content/plugins/wp-filebase/extras/jquery/treeview/jquery.treeview.async.js?ver=0.3.0.06:48:6)
    at i (http://172.20.10.2/portal/wp-includes/js/jquery/jquery.js?ver=1.12.4:2:27449)
    at Object.fireWith [as resolveWith] (http://172.20.10.2/portal/wp-includes/js/jquery/jquery.js?ver=1.12.4:2:28213)
ca @ jquery.js?ver=1.12.4:3
ja @ jquery.js?ver=1.12.4:3
Ha @ jquery.js?ver=1.12.4:3
append @ jquery.js?ver=1.12.4:3
n.fn.(anonymous function) @ jquery.js?ver=1.12.4:3
createNode @ jquery.treeview.async.js?ver=0.3.0.06:15
each @ jquery.js?ver=1.12.4:2
success @ jquery.treeview.async.js?ver=0.3.0.06:48
i @ jquery.js?ver=1.12.4:2
fireWith @ jquery.js?ver=1.12.4:2
y @ jquery.js?ver=1.12.4:4
c @ jquery.js?ver=1.12.4:4
XMLHttpRequest.send (async)
send @ jquery.js?ver=1.12.4:4
ajax @ jquery.js?ver=1.12.4:4
load @ jquery.treeview.async.js?ver=0.3.0.06:40
toggle @ jquery.treeview.async.js?ver=0.3.0.06:103
settings.toggle @ jquery.treeview.js?ver=0.3.0.06:104
d.complete @ jquery.js?ver=1.12.4:4
i @ jquery.js?ver=1.12.4:2
fireWith @ jquery.js?ver=1.12.4:2
i @ jquery.js?ver=1.12.4:4
n.fx.tick @ jquery.js?ver=1.12.4:4
setInterval (async)
n.fx.start @ jquery.js?ver=1.12.4:4
n.fx.timer @ jquery.js?ver=1.12.4:4
qb @ jquery.js?ver=1.12.4:4
g @ jquery.js?ver=1.12.4:4
dequeue @ jquery.js?ver=1.12.4:3
(anonymous) @ jquery.js?ver=1.12.4:3
each @ jquery.js?ver=1.12.4:2
each @ jquery.js?ver=1.12.4:2
queue @ jquery.js?ver=1.12.4:3
animate @ jquery.js?ver=1.12.4:4
heightToggle @ jquery.treeview.js?ver=0.3.0.06:34
toggler @ jquery.treeview.js?ver=0.3.0.06:145
dispatch @ jquery.js?ver=1.12.4:3
r.handle @ jquery.js?ver=1.12.4:3

Any ideas?

thanks a lot.
GR.

**Comment by jepsonrob** [comment]
I have the same problem, I'm currently playing around with different versions of wordpress and filebase to see if anything fixes it. Have you come across any solution to this?

**Comment by jepsonrob** [comment]
Yeah it didn't work for the latest (4.9.4) or 4.5.2 but it works fine for version 4.3.1.
Hope this helps!
